fix down spelling error

dfuentes77 opened the pull request:
### Description

fixes misspelling of `down` in output of `chef-server-ctl stop`

### Definition of Done

it says `down` instead of `diwb`

### Check List

- [X] Spell Check

Signed-off-by: Damian Fuentes <damian@cerego.com>

dfuentes77 committed:
fix down spelling error

tas50 commented:
@dfuentes77 can you sign off this PR for the DCO? There's more information on why we require it here: https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco

dfuentes77 commented:
I added my signoff to the comment.  Is that good enough?

Commit:
Merge branch 'master' into patch-1

tas50 committed:
Improve knife plugin page titles (#2030)

* Improve knife plugin page titles

Make it a bit more obvious what's in these pages and nuke some bad information.

Signed-off-by: Tim Smith <tsmith@chef.io>

IanMadd committed:
Merge two identical dtags & add Attributes heading (#2021)

* merge dtags resources_common_guards_attributes & resources_common_guards_properties. Add Attributes heading.

* Replace attributes with properties

* Replace two more instances of Attributes with Properties

Signed-off-by: IanMadd <imaddaus@chef.io>

Commit:
Merge branch 'patch-1' of https://github.com/dfuentes77/chef-web-docs into im/fix_down

Commit:
Fix dtags

Signed-off-by: IanMadd <imaddaus@chef.io>

IanMadd reviewed (approved):
![tenor-189932832](https://user-images.githubusercontent.com/8138169/61645358-6c9e2c80-ac74-11e9-81a3-b47ab995743e.gif)
